src/repositorios/colllection.test.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
```

Comment:
File name has a typo. Rename it from `src/repositorios/colllection.test.js` to `src/repositorios/collection.test.js`.